pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -336,10 +336,6 @@ stdenv.mkDerivation ({
 
     echo configureFlags: $configureFlags
     ${setupCommand} configure $configureFlags 2>&1 | ${coreutils}/bin/tee "$NIX_BUILD_TOP/cabal-configure.log"
-    if ${gnugrep}/bin/egrep -q -z 'Warning:.*depends on multiple versions' "$NIX_BUILD_TOP/cabal-configure.log"; then
```

Comment:
I think you've removed the only call to `gnugrep` so you can remove it from the `inherit`?